### PR TITLE
same class name

### PR DESCRIPTION
**Pull request by tael**
fix this error.
-----
The class "FOS\RestBundle\View\View" is not annotated with @Annotation. Are you sure this class can be used as annotation? If so, then you need to add @Annotation to the _class_ doc comment of "FOS\RestBundle\View\View".

### CODE REVIEW

**Comment by Ener-Getick** [comment]
That's an error in the use statement, the annotation is https://github.com/FriendsOfSymfony/FOSRestBundle/blob/master/Controller/Annotations/View.php.

**Comment by tael** [comment]
@Ener-Getick 

I think this example is not possible.
As you know, `$view = View::create()`  is not using Annotations/View but Class View
(https://github.com/FriendsOfSymfony/FOSRestBundle/blob/master/View/View.php)

Could you check this example again?
Also, you can check this example :D
https://github.com/liip/LiipHelloBundle/blob/master/Controller/ExtraController.php

thanks

```
use FOS\RestBundle\View\View;

/**
 * @View()
 */
public function getUsersAction()
{
    $view = View::create();
// ...
    return $view;
}
```

**Comment by Ener-Getick** [comment]
Well in fact both classes are used. View\View in the method content and Annotations\View for the annotation, an alias should be used here :)

**Comment by tael** [comment]
Oh, really? 
Maybe.. I missed somthing... :(
Can you explain what `@View()` doing? Or please leave some links.
Anyway, if it's correct, 
the code need two use statements, right?

**Comment by Ener-Getick** [comment]
@tael it allows to omit the View::create() for those who prefer annotations.
And yes we need two use statements :)

**Comment by tael** [comment]
I'm not familiar with pull requests and english, sorry :(
Anyway, this thread  is done! maybe..
any action is needed for me?

**Comment by Ener-Getick** [comment]
@tael no problem you can ask questions if you don't know how to do something 😉

File: Resources/doc/view_response_listener.rst [hunk 1]
```diff
@@ -148,15 +148,16 @@ You can also define your serializer options dynamically:
 
     <?php
 
-    use FOS\RestBundle\View\View;
+    use FOS\RestBundle\Controller\Annotations\View;
```
Comment (Ener-Getick) [inline Resources/doc/view_response_listener.rst]: I usually prefer annotating annotations. What about `ViewAnnotation`?

File: Resources/doc/view_response_listener.rst [hunk 2]
```diff
@@ -148,15 +148,16 @@ You can also define your serializer options dynamically:
...
+    use FOS\RestBundle\View\View as ViewClass;
     use FOS\RestBundle\Context\Context;
 
-    /**
-     * @View()
-     */
+    /**		
+     * @View()		
+     */
```
Comment (Ener-Getick) [inline Resources/doc/view_response_listener.rst]: Can you remove those spaces please ?

File: Resources/doc/view_response_listener.rst (hunk 1 above)
Comment (tael) [inline Resources/doc/view_response_listener.rst]: I agree using `ViewAnnotation`.
but `View` on this document  ( http://symfony.com/doc/current/bundles/FOSRestBundle/view_response_listener.html )
using Annotation globally, so I think remains that for consistency.
what do you think about, @Ener-Getick ?

Comment (Ener-Getick) [inline Resources/doc/view_response_listener.rst]: Looks good to me 👍

**Comment by tael** [comment]
@Ener-Getick 
I want to finish this PR with no problem.
So... if I missing something to do more, please tell me again :D 

* Who can close this thread? me? or any reviewers? 
(Oh, if this message makes hurry, sorry. I just want to check my mistake :D)

**Comment by Ener-Getick** [comment]
@tael no problem, my comment https://github.com/FriendsOfSymfony/FOSRestBundle/pull/1553#discussion-diff-75634744 still isn't fixed :)

**Comment by Ener-Getick** [comment]
Thank you @tael ! :)

**Comment by tael** [comment]
NP @Ener-Getick :D 
I need to improve my english and github expriences ;D
Anyway, it's good to me, thank you again @Ener-Getick .

**Comment by Ener-Getick** [comment]
@tael if you want to fix bugs or propose features i advise you to take a look at the symfony repository, it is much bigger and there are always things moving ;)
You can also review the other PRs to practice english (even if most of us don't have a perfect english :P).

**Comment by tael** [comment]
@Ener-Getick thanks for your advices! I will look around symfony 👍 
I remember that  simmilar PR you closed. ( https://github.com/FriendsOfSymfony/FOSRestBundle/pull/1554 ) 
is it merged already? Or can i PR again?

**Comment by Ener-Getick** [comment]
That's the same except that's targeting another branch, right ?
We're usually merging branchs in each other ourselves unless there are too complex conflicts.

**Pull request merged**